gtest/src/manager.rs

```diff
@@ -1055,6 +1186,14 @@ impl JournalHandler for ExtManager {
 
         let source = dispatch.source();
 
+        if dispatch.value() != 0 {
```

Comment:
must not deposit when message is emitted within event

Reply:
Fixed